demo-app/src/components/RenderFromFile.tsx

```diff
@@ -30,8 +22,9 @@ export default async function RenderFromFile({
     }
   }
 
-  const parsedConfig = parseYaml(config)
-  const renderedComponents = applyConfiguration(components)(parsedConfig, {})
+  if (config) {
+    return <RenderConfig config={config} />
+  }
 
-  return <>{renderedComponents}</>
+  return <div>No config :(</div>
```

Comment:
Kinda the same thing here, change the if to `!config` and then make the default return be RenderConfig.